js/globals.ts

```diff
@@ -6,7 +6,7 @@ import { globalEval } from "./global_eval";
 import { libdeno } from "./libdeno";
 import * as textEncoding from "./text_encoding";
 import * as timers from "./timers";
-import * as urlSearchParams from "./url_search_params";
+import * as urlSearchParams_ from "./url_search_params";
```

Comment:
you don't need to underscore this... it is only needed when there is a name collision in the global namespace, (e.g. `console` or `fetch`).  Since there is no global variable named `urlSearchParams` it is safe to use.

Reply:
My bad, I just thought it was the same as the browser.